src/prebid.js

```diff
@@ -110,6 +120,61 @@ function init(timeout, adUnitCodeArr) {
 	}
 }
 
+/*
+ *	get Sizes by viewable area size
+ */
+function getSizeByDynamicSizes(dynamicSizes){
```

Comment:
@hjeong12  

These private functions shouldn't be in prebid.js. Can you move to utils.js?